src/mono/wasm/build/WasmApp.targets

```diff
@@ -211,4 +249,24 @@
       <WasmAssembliesFinal Include="@(_WasmAssembliesInternal)" LlvmBitCodeFile="" />
     </ItemGroup>
   </Target>
+
+  <!-- Setup NPM packages for NodeJS publish/builds -->
+  <Target Name="_NodePrep" AfterTargets="_AfterWasmBuildApp" Condition="'$(ForNode)' == 'true'">
+    <!-- If package.json is in the directory copy it to new output directory -->
+    <Copy SourceFiles="package.json"
+        DestinationFolder="$(WasmAppDir)"
+        SkipUnchangedFiles="true" 
+        Condition="Exists('package.json')"/>
+
+    <!-- Otherwise copy the default template into the output directory -->
+    <Copy SourceFiles="$(MSBuildThisFileDirectory)\template-package.json"
+        DestinationFiles="$(WasmAppDir)\package.json"
+        SkipUnchangedFiles="true" 
+        Condition="!Exists('package.json')"/>
+    
+    <!-- npm install -->
+    <Exec Command="npm install"
```

Comment:
I would prefer to use `npm ci` and include lock file. We do not want npm to install something nasty.
See https://docs.npmjs.com/cli/v7/commands/npm-ci

Reply:
I just tried it but I guess `npm ci` is not defined on this ancient version of npm that is on CI

Reply:
I'm going to try to use the npm defined as part of emsdk